Assets/Scripts/Controllers/CharacterController-DESKTOP-7IQNCQ4.cs

```diff
@@ -7,13 +7,16 @@ public class CharacterController : MonoBehaviour
 {
     private Rigidbody2D _rb;
     private Vector2 _movement;
+    private CapsuleCollider2D _cc;
```

Comment:
`_rb` is a fairly standard abbreviation for `_rigidbody`, so that's fine. `_cc` is pretty vague though.

Request changing this to something more descriptive, like `_characterCollider`.

Hint: In Visual Studio, do the following to do a _Replace By Symbol_:
- Select `_cc`
- Press `Ctrl+R+R`
- Type in the new name
- Hit Enter to rename the symbol in all places within the solution.